src/main/webapp/profileScript.js

```diff
@@ -546,3 +553,15 @@ function showNameForm() {
   localStorage.setItem(LOCAL_STORAGE_NAME, 'justChanged');
   document.getElementById('name-form-container').style.display = 'block';
 }
+
+/*
+ * Presents the user with a form to change their profile picture.
```

Comment:
Do you have somewhere to delete the image stored in blobstore previously? It looks like right now when the user uploads a new profile picture their old one is just floating around in blobstore space, and we don't really have a way to access it since we no longer have the blobkey (technically you can delete the blob from the blobstore admin page on the deployed version so it isn't really GONE but you can't access it from the code).